README.md

```diff
@@ -6,4 +6,4 @@ When a statue arrives broken what do you normally do, you request either a refun
 2. They refuse point blank to give a refund. 
 3. They do issue a refund. 
 
-Main thing to ask, is it the seller fault when a Product comes broken and quiet simiply yes it is no one pays for a broken statue. 
```

Comment:
Main thing to ask is this. It is the sellers fault when a product comes broken and quiet simiply the answer is yes. No one pays for a broken statue. Unless it was listed as such before the sale.